src/utils/types.js

```diff
@@ -8,6 +8,7 @@ type FlowAnyType = { type: "any", nullable: boolean };
 
 export type FlowScalarType = FlowBooleanType | FlowNumberType | FlowStringType | FlowAnyType;
 export type FlowObjectType = { type: "object", nullable: boolean, properties: FlowTypes };
-export type FlowType = FlowObjectType | FlowScalarType;
+export type FlowArrayType = { type: "array", nullable: boolean, children: FlowTypes | null };
```

Comment:
I thought I tried that and it yelled at me, but maybe it was related to something else.